syftlib/src/main/java/org/openmined/syft/execution/JobStatusMessage.kt

```diff
@@ -7,9 +7,12 @@ import java.util.concurrent.ConcurrentHashMap
 @ExperimentalUnsignedTypes
 sealed class JobStatusMessage {
     class JobCycleRejected(val timeout: String) : JobStatusMessage()
-    class JobReady(
+
+    class JobJobReady(
```

Comment:
`JobJobReady` can lead to confusion. I mean... I'm confused :)

Reply:
Oh that's a typo